### PR TITLE
Removes integrated circuits wirer

### PR DESCRIPTION
**Pull request by ACCount12**
Circuit wirer seems like an essential tool until you realize that a regular multitool does the same exact thing, and it does it way better. Say "no" to tool clutter!

:cl: ACCount
del: Removed integrated integrated circuits wirer. Multitool has the same functionality anyway.
/:cl:

### CODE REVIEW

**Comment by WJohn** [comment]
Now do this with all the other tools and give our existing sets more uses. Maybe the regular old air analyzer could finally have a useful purpose?

**Comment by ACCount12** [comment]
It's used for anomaly hunting, and any engineer knows how useful it is for its primary purpose.

**Comment by WJohn** [comment]
Yeah but it can always have MORE purposes than basically just those two, no? Our tools are all very useful but the analyzer has always been something I feel like throwing out because who even cares about scanning the air when there are air alarms literally everywhere that do it for you and a lot of PDAs on the station already have that function?

**Comment by Bawhoppen** [comment]
@WJohn Merging different types of tools into one is bad. Stuff should have specific unique purposes.

**Comment by ACCount12** [comment]
Merging different types of tools into one is good. We really shouldn't have snowflake items for any conceivable purpose.

I plan to make IC debugger a subtype of multitool and make slime scanner a subtype of health scanner. It would be better for items like this to extend basic functionality instead of being barely used snowflakes.

**Comment by Bawhoppen** [comment]
Removing any uniqueness just degrades the game even deeper into another mechanical-based match-off, which lacks any intrigue or dynamic liveliness.

**Comment by ACCount12** [comment]
Sorry, but this excuse is "sense of pride and accomplishment" tier.

File: code/modules/integrated_electronics/core/integrated_circuit.dm
Comment (Cyberboss) [inline code/modules/integrated_electronics/core/integrated_circuit.dm]: just `in assembly`

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Bawhoppen** [comment]
Literally no argument for removing them other than dumbing things down

**Comment by davethwave** [comment]
All I can say is how does the multitool even wire and rewire stuff in a circuit? The multitool is for testing wires and pulsing mostly for hacking and has some functionality in telecoms. The sprite that the multitool has would be out of place as something for integrated circuits and would bug me to no end as it doesn't seem to have a way to make changes to a circuit visibly in its sprite as a wiring tool. If I were doing this I would perhaps replace the debugger or analyzer instead as with the multitool as it seems to stay more in theme to what the multitool does.

**Comment by WJohn** [comment]
item clutter and bloat is awful. an item with literally one minor use which is also already made redundant in another item that does it too, AND has far more uses on top of that means there's no reason to keep it.

why on earth would use want to keep the circuits wirer that is only capable of doing wiring, when you can have the multitool which can do it as well and also has tons of hacking utility in pretty much anything with a panel and cables?

**Comment by jughu** [comment]
This is for when you dont have a multitool and hand lmfao

File: code/modules/integrated_electronics/core/integrated_circuit.dm
Comment (ACCount12) [reply code/modules/integrated_electronics/core/integrated_circuit.dm]: I'm rewriting this completely in another PR.

**Comment by ACCount12** [comment]
It would make sense for, you know, wires and wirecutters to do the wiring and rewiring. But assemblies don't even use the same wires type.

**Comment by Bawhoppen** [comment]
I just don't see how anyone could support stuff like this for anything other than the absolutely most meager amount of performance boost from removing 100 lines of code

**Comment by kevinz000** [comment]
alternatively we can disable multitool wiring :^)

**Comment by arsserpentarium** [comment]
maybe just leave it as it is?
Wire with multitool,or wirer is questio of personal choice.Don't like wirer? Don't print it!

**Comment by ACCount12** [comment]
On hold until my other IC pull request gets merged.

**Pull request closed without merging**
